content/docs/5.1/reference/faq.md

```diff
@@ -9,141 +9,143 @@ menu:
+TiKV has an optimistic transaction model, which means that the client buffers all writes in the transaction. When the client calls the commit function, the writes are packed and sent to the server. If there are no conflicts, the key-value pair writes with specific version information are written to the database and can be read by other transactions.
 
 ## Can I use TiKV as a key-value store?
 
-Yes. That's what TiKV is.
+Yes. That is what TiKV is.
 
 ## How does TiKV compare to NoSQL databases like Cassandra, HBase, or MongoDB?
 
-TiKV is as scalable as NoSQL databases like to advertise, while including features like externally consistent distributed transactions and good support for stateless query layers such as TiSpark (Spark), Titan (Redis), or TiDB (MySQL)
+TiKV has the scalability as NoSQL databases. At the same time, it also has the strong consistency for distributed transactions and provides solid support for the stateless query layers, such as TiSpark (Spark), Titan (Redis), or TiDB (MySQL).
 
-## What is the recommended number of replicas in the TiKV cluster? Is it better to keep the minimum number for high availability?
+## How many replicas are recommended to be set in a TiKV cluster? Is it better to keep the minimum number of replicas in a TiKV cluster for TiKV's high availability?
 
-3 replicas for each Region is sufficient for a testing environment. However, you should never operate a TiKV cluster with under 3 nodes in a production scenario. Depending on infrastructure, workload, and resiliency needs, you may wish to increase this number.
+Three replicas for each Region is sufficient for a testing environment. However, you must not operate a TiKV cluster with less than three nodes in a production environment. Depending on the infrastructure, workload, and resiliency needs, you can increase the number as you need.
 
-## If a node is down, will the service be affected? How long?
+## If a node is down, will the service be affected? If so, how long will it take?
 
-TiKV uses Raft to synchronize data among multiple replicas (by default 3 replicas for each Region). If one replica goes wrong, the other replicas can guarantee data safety. Based on the Raft protocol, if a single leader fails as the node goes down, a follower in another node is soon elected as the Region leader after a maximum of 2 * lease time (lease time is 10 seconds).
+TiKV uses Raft to synchronize data among multiple replicas (three replicas for each Region by default). If one replica is down, the data safety can still be guaranteed by other replicas. Based on the Raft protocol, once a node is down, and a single leader fails, a follower in another node is elected as the Region leader right after. This substitution process takes no longer than 2 * lease time (lease time is 10 seconds).
```

Comment:
> Active voice is preferred.

since we try to avoid using "will“ in cases that certainly happen, I used "is" instead of "will". I think we can change the sentence sequences at your suggestion.